### PR TITLE
Feature/roi transfer

### PR DESCRIPTION
**Pull request by tdlinh2712**

### CODE REVIEW

File: src/Model/MovingModel.py [hunk 1]
```diff
@@ -173,9 +173,10 @@ def read_images_for_fusion(level=0, window=0):
...
 
     create_fused_model(orig_image, new_image)
-    color_axial, color_sagittal, color_coronal = \
+    color_axial, color_sagittal, color_coronal, tfm = \
         get_fused_window(level, window)
 
     patient_dict_container.set("color_axial", color_axial)
     patient_dict_container.set("color_sagittal", color_sagittal)
     patient_dict_container.set("color_coronal", color_coronal)
+    moving_dict_container.set("tfm", tfm)
```
Comment (DuyTungHa) [inline src/Model/MovingModel.py]: PEP8 - end of file

File: src/View/util/PatientDictContainerHelper.py [hunk 2]
```diff
@@ -0,0 +1,4 @@
+
+def get_dict_slice_to_uid(patient_dict_container):
+    return dict(
+        (v, k) for k, v in patient_dict_container.get("dict_uid").items())
```
Comment (DuyTungHa) [inline src/View/util/PatientDictContainerHelper.py]: PEP-8 end of file

**Review by DuyTungHa — COMMENTED** [review]
Very weird behavior of image fusion window (please see images)
image
image

**Comment by tdlinh2712** [comment]
> Very weird behavior of image fusion window (please see images)
> image
> image

does this happen in master as well? or just on this branch

**Review by DuyTungHa — COMMENTED** [review]
1. Progress Window too small
image
2. Why does Image Fusion Window always load default directory and not the previous directory ? 
3. There are some DEBUG logs in the terminal when doing ROI Transfer.

**Pull request closed without merging**